playbook/app/pb_kits/playbook/pb_checkbox/docs/_checkbox_indeterminate.html.erb

```diff
@@ -0,0 +1,77 @@
+<%= pb_rails("table", props: { size: "md" }) do %>
+  <thead>
+    <tr>
+      <th>
+        <%= pb_rails("checkbox" , props: {
+            text: "Select ",
+            value: "checkbox-value",
+            name: "main",
+            indeterminate: true,
+            id: "test-indeterminate-js"
+        }) %>
+        </th>
+    </tr>
+  </thead>
+  <tbody>
+    <tr>
+      <td>
+        <%= pb_rails("checkbox" , props: {
+          text: "Item 1",
+          value: "checkbox-value",
+          checked: true,
+          name: "checkbox-name",
+          classname: "check-js"
+        }) %>
+      </td>
+    </tr>
+    <tr>
+      <td>
+        <%= pb_rails("checkbox" , props: {
+          text: "Item 2",
+          value: "checkbox-value",
+          checked: false,
+          name: "checkbox-name",
+          classname: "check-js"
+        }) %>
+      </td>
+    </tr>
+  </tbody>
+<% end %>
+
+
+<script>
+  const indeterminateCheckbox = document.querySelector("#test-indeterminate-js");
```

Comment:
This kit seems like a good candidate to reuse the react kit

Reply:
I agree. We will have to get the form situation worked out to ensure that anyone using this kit will receive the same markup as expected by Rails within at least the `form_for` implementation. Some work is necessary to make that happen within the form kit itself.